### ISSUE TITLE
Add with-graphql example

### ISSUE THREAD

**Issue by rishabhpoddar** [supertokens/docs#4]
Once this is done, reply to https://www.reddit.com/r/flask/comments/nsxuxa/user_authentication_with_graphql_how_the_fuck/

**Comment by rishabhpoddar** [comment]
Closing in favour of https://github.com/supertokens/for-zenhub/issues/91
